src/use_case/statistics/StatisticsInteractor.java

```diff
@@ -0,0 +1,29 @@
+package use_case.statistics;
+
```

Comment:
```suggestion
import java.util.HashMap;
```
Sorry, for some reason this wasn't added in the code suggestion, my appologies